support.md

```diff
@@ -11,24 +11,31 @@ The official Petals documentation is available at [doc.petalslink.com](https://d
 
 # White Papers
 
-The Petals team has written several white papers.
+The Petals team has written several white papers:  
 
-* [A series of articles](https://blog.linagora.com/vision-des-esbs-par-un-architecte-dentreprise-ou-ce-que-devraient-etre-les-esbs/) written by an Enterprise Architect (in French).
-* In French: [Petals ESB, un bus de services](resources/pdf/Livre_blanc_ESB_Petals.pdf)
-* [How to choose your process orchestration technology?](resources/pdf/white_paper_orchestration.pdf)
+* [A series of articles](https://blog.linagora.com/vision-des-esbs-par-un-architecte-dentreprise-ou-ce-que-devraient-etre-les-esbs){:target="_blank"} written by an Enterprise Architect (in French).
+* In French: [Petals ESB, un bus de services](resources/pdf/Livre_blanc_ESB_Petals.pdf){:target="_blank"}
+* [How to choose your process orchestration technology?](resources/pdf/white_paper_orchestration.pdf){:target="_blank"}
 
-Various presentations are also available on [Slide Share](https://slideshare.net/petalslink).
+Various presentations are also available on [Slide Share](https://slideshare.net/petalslink){:target="_blank"}.
+
+And somes videos conferences are also available :
```

Comment:
"And some video conferences are also available"

Reply:
"And some video conferences are available".
Avoid and + also together. :)